locales/en-GB/messages.properties

```diff
@@ -86,16 +86,16 @@ fileSavingIndicator=Saving...
 fileSavingFailedIndicator=Saving failed, retrying...
 help=Help
 deleteProjectLink=Delete Project
-exportProjectLink=Export (.zip)
+exportProjectLink=Download files (.zip)
```

Comment:
Correct! @simon66 this change should be made in the properties file in `en-US` not `en-GB`